src/Transport/Sending/DefaultOutgoingBatchRouter.cs

```diff
@@ -118,11 +117,16 @@ string GetViaEntityPathFor(EntityInfo entity)
             return null;
         }
 
-        async Task RouteOutBatchesWithFallbackAndLogExceptionsAsync(IMessageSender messageSender, IList<IMessageSender> fallbacks, IList<BrokeredMessage> messagesToSend, BrokeredMessageReceiveContext context, bool useTransaction)
+        async Task RouteOutBatchesWithFallbackAndLogExceptionsAsync(IMessageSender messageSender, IList<IMessageSender> fallbacks, IList<BrokeredMessage> messagesToSend,bool suppressTransaction)
         {
             try
             {
-                await RouteBatchWithEnforcedBatchSizeAsync(messageSender, messagesToSend, context, useTransaction).ConfigureAwait(false);
+                var scope = suppressTransaction ? new TransactionScope(TransactionScopeOption.Suppress, TransactionScopeAsyncFlowOption.Enabled) : null;
```

Comment:
Do we need to set IsolationLevel to Serializablie just like [here](https://github.com/Particular/NServiceBus.AzureServiceBus/pull/299/files#diff-446ccf5d2a05a3320ebaf7992c901021R183)?